docs/ConverterUsage.md

```diff
@@ -0,0 +1,98 @@
+# Converter Usage & Demo
+
+## Usage
+
+`jfrconv [options] <input> [<input>...] <output>`
+
+The output format specified can be only one at a time for conversion from one format to another.
+
+### Available arguments
+
+```
+Conversion options: 
+  -o --output FORMAT   Output format: <filename> followed by any of the extensions: html, collapsed, pprof, pb.gz
+  
+JFR options:
+    --cpu              Generate only CPU profile during conversion
+    --wall             Generate only Wall clock profile during conversion
+    --alloc            Generate only Allocation profile during conversion
+    --live             Build allocation profile from live objects only during conversion
+    --lock             Generate only Lock contention profile during conversion
+ -t --threads          Split stack traces by threads
+ -s --state LIST       Filter thread states: runnable, sleeping
+    --classify         Classify samples into predefined categories
+    --total            Accumulate total value (time, bytes, etc.) instead of samples
+    --lines            Show line numbers
+    --bci              Show bytecode indices
+    --simple           Simple class names instead of fully qualified names
+    --norm             Normalize names of hidden classes/lambdas
+    --dot              Dotted class names instead of / notation
```

Comment:
Better with an example: Dotted class names, e.g. `java.lang.String` instead of `java/lang/String`.